hawkular-metrics/Dockerfile

```diff
@@ -77,10 +77,10 @@ COPY logging.properties $JBOSS_HOME/standalone/configuration/logging.properties
 
 # Install Hawkular-Metrics Python client, only available on EPEL
 USER root
-RUN yum -y install epel-release \
-    && yum -y install python-pip \
-    && yum clean all \
-    && pip install --no-cache-dir 'hawkular-client==0.4.4'
+#RUN yum -y install epel-release \
+#    && yum -y install python-pip \
+#    && yum clean all \
+#    && pip install --no-cache-dir 'hawkular-client==0.4.4'
```

Comment:
Please don't comment this section out of the code. We still want to have the client installed